src/config.ts

```diff
@@ -0,0 +1,3 @@
+export const applicationConfig = {
+  apiUrl: process.env.REACT_APP_API_URL,
+};
```

Comment:
👀 is this like a "nice"/"neat" way of doing env vars, and should we do the same with `AAD_CLIENT_ID`?

Reply:
Oh, I didn't see your auth stuff actually!  It probably makes sense to have everything in one place from an app config perspective so it's easier to find?

I can put the client id in this file and then reference it from the `AuthWrapper`?

Reply:
Yeah, that sounds great! :D

Reply:
Updated to use the application config 😄